docs/tools-reference/linux-users-and-groups/index.md

```diff
@@ -12,7 +12,7 @@ modified_by:
 published: 2009-08-31
 title: Linux Users and Groups
 external_resources:
- - '[User Account and Group Management @ UWISC''s Center for Computer Aided Engineering](http://www.cae.wisc.edu/linaccounts)'
```

Comment:
Broken link?

Reply:
@bagasme Yes but can't find the content on the page anywhere else, even

Reply:
After reviewing [web.archive.org snapshot](http://web.archive.org/web/20160926055501/http://www.cae.wisc.edu/cae-account-management/), actually it is account management home page for CAE course, not related to our Linux Users & Groups guide.

So removing this external resource is correct, to keep links related.

Reply:
Indeed, the external link above bounce to course account management page.

Reply:
Edit: It is actually broken link without redirection

Reply:
right